frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadList.component.tsx

```diff
@@ -33,19 +33,22 @@ export const UploadList = ({
 	onClickDelete,
 }: IUploadList): JSX.Element => {
 	const { trigger, setValue } = useFormContext();
+	const [selectedListItem, setSelectedListItem] = useState(null);
 	return (
 		<Container>
 			{
 				values.map((item, index) => (
 					<UploadListItem
 						key={item.uploadId}
 						item={item}
+						onClickRow={() => setSelectedListItem(index)}
```

Comment:
If Im not mistaken the selection is only when the item is being edited so it wouldn't be triggered by clicking a row.
So the row would be selected only in the case that the side panel is opened
I think that's what @george-hadfield said. Correct me if Im wrong.